src/main/resources/assets/schemas/Study.yaml

```diff
@@ -27,6 +27,11 @@ properties:
       type: integer
       format: int32
     description: The ids of all of the datasets related to this study.
+  datasets:
```

Comment:
👍🏽  This is what I think the name should be. What I see when calling this is the field name defined in `Study` ...
```
...
  "datasetIds": [
    1427
  ],
  "datasetSet": [
    {
      "dataSetId": 1427,
...
```